netutils/netcat/Kconfig

```diff
@@ -0,0 +1,52 @@
+#############################################################################
+#
+# netutils/netcat/Kconfig
+# netcat networking application
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+#############################################################################
+
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config NETUTILS_NETCAT
+	bool "NetCat tool"
+	default n
+	select NET
+	select NET_TCP
```

Comment:
should we change to depends on NET_TCP? since all other place use depend not select.

Reply:
both would work , I can align to others and we can change all in a other changes if needed